src/sentry/plugins/migrator.py

```diff
@@ -25,37 +31,42 @@ def call(self):
                     # can disable the Plugin for all Projects. There'd be no
                     # Repos left, associated with the Plugin.
                     self.disable_for_all_projects(plugin)
+        self._logging_context()
```

Comment:
I don't think this will preserve the existing behavior. Previously the attributes in `_logging_context` would be added as context data to all the logs that mediators make. We could retain that by using `_logging_context()` to create logging context data.

Reply:
ohhh I see,  yeah I completely misunderstood that context func.